Samples/PipeableSample/PipeableSample/PipeableWebViewWrapper.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+import PipeableSDK
+import SwiftUI
+import WebKit
+
+struct PipeableWebViewWrapper: UIViewRepresentable {
+    @Binding var items: [NewsItem]
+    var onClose: () -> Void
+
+    func makeUIView(context _: Context) -> PipeableWebView {
+        let webView = PipeableWebView()
+        let page = webView.page
+
+        Task {
+            _ = try? await page.goto("https://news.ycombinator.com", waitUntil: .networkidle)
+
+            try? await sleep(ms: 1000)
+
+            let searchForm = try await page.waitForXPath(
+                "//form[contains(@action, 'hn.algolia.com')]",
+                visible: true
+            )
+
+            guard let textAreaEl = try await searchForm?.querySelector("input[type='text']") else {
+                return
+            }
+
+            _ = try await page.evaluateAsyncFunction(
+                """
+                    el.scrollIntoView();
+
+                    el.focus();
+
+                    await new Promise((res) => setTimeout(res, 500));
+
+                    // Scroll a bit to the top, so we can visualize what we are typing.
+
+                    window.scrollTo(0, window.scrollY + 300);
+                """, arguments: ["el": textAreaEl]
+            )
+
+            try? await sleep(ms: 1000)
+
+            try await textAreaEl.type("web automation", delay: 50)
+
+            _ = try await page.submitActiveForm()
+
+            try await page.waitForURL { url in url.contains("hn.algolia") }
+
+            _ = try await page.waitForSelector(".Story_title", visible: true)
+
+            // Hold a bit, so it's visible.
+            try? await sleep(ms: 2000)
```

Comment:
Is this for the demo to show what is going on or is this required for the automation to work?

Reply:
Just for demo purposes. I'll tweak the comment, so it's clear.